blog/deployment.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: blog
+  labels:
+    app: blog
+spec:
+  serviceName: blog
+  replicas: 1
+  selector:
+    matchLabels:
+      app: blog
+  template:
+    metadata:
+      labels:
+        app: blog
+    spec:
+{% if deploy %}
+      priorityClassName: production
+{% endif %}
+      containers:
+      - name: blog
+        image: ghost:3.0-alpine
+        imagePullPolicy: Always
```

Comment:
Is there a reason for this to be Always over IfNotPresent?

Reply:
I don't believe so. I can change it; should I just remove that line and have it use the default?

Reply:
yep, removing it is the right action, default is IfNotPresent